### ISSUE TITLE
enum in message object

### ISSUE THREAD

**Issue by AndreasMadsen** [mafintosh/protocol-buffers#26]
In the riak protocol buffer defintion they have an `enum` inside a `message` scheme. (see: https://github.com/basho/riak_pb/blob/master/src/riak_kv.proto#L160). However in this module there do not appear to be a way to get that `enum` dictionary/object. 

```javascript
var protobuf = require('protocol-buffers');

var source = function () { /*
// Secondary Index query request
message RpbIndexReq {
  enum IndexQueryType {
    eq = 0;
    range = 1;
  }

  required bytes bucket = 1;
  required bytes index = 2;
  required IndexQueryType qtype = 3;
  optional bytes key = 4; // key here means equals value for index?
  optional bytes range_min = 5;
  optional bytes range_max = 6;
  optional bool return_terms = 7;
  optional bool stream = 8;
  optional uint32 max_results = 9;
  optional bytes continuation = 10;
  optional uint32 timeout = 11;
  optional bytes type = 12;         // Bucket type, if not set we assume the 'default' type
  optional bytes term_regex = 13;
  // Whether to use pagination sort for non-paginated queries
  optional bool pagination_sort = 14;
}
*/}.toString().split('\n').slice(1, -1).join('\n');

// Parse and export messages
var messages = protobuf(source);

console.log(messages.RpbIndexReq.IndexQueryType); // undefined
console.log(messages.IndexQueryType); // undefined
```

**Comment by kevincennis** [comment]
Ran into the same thing. Fixed in #30.

**Comment by mafintosh** [comment]
Yes. Fixed by #30 and released in 2.5.0
